functions_test.go

```diff
@@ -160,13 +159,13 @@ func (*functionSuite) TestCause(c *gc.C) {
 	// Look for a file that we know isn't there.
 	dir := c.MkDir()
 	_, err = os.Stat(filepath.Join(dir, "not-there"))
-	c.Assert(os.IsNotExist(err), jc.IsTrue)
+	c.Assert(os.IsNotExist(err), IsTrue)
```

Comment:
Instead of adding a whole bunch of extra code for the checker, we can just do
`c.Assert(blah, gc.Equals, true)`

Reply:
Good point